test/test_get_account_statement.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pytest
+
+import pytransferwise
+
+pytransferwise.api_key = os.getenv('TW_API_KEY')
```

Comment:
Nit: just for clarity, could we call this environment variable `TRANSFERWISE_API_KEY`?

If you could update the [README test docs](https://github.com/jayaddison/pytransferwise#run-tests) to let users to know to specify a sandbox key that would be useful too.

(sidenote: we should probably break this out into a separate suite of integration tests, since they require network connectivity to run -- but that can be done outside of this pull request)

Reply:
@jayaddison , this is done too. I agree the integration tests better to put separetly. Also because the future ones will require the test data preparation. Even this one smoke one need to create at least profile and account in sandbox.